### PR TITLE
Wikipedia API change + search_config.json parsing error fix

### PR DESCRIPTION
**Pull request by arpitaagrawal**
@chrismattmann - Committing the Wikipedia API change. Also, the search-config.json had a json parsing error - a ',' was missing in the json because of which the search of words was breaking in the existing code. Have rectified that and have committed the same as well.

### CODE REVIEW

**Comment by chrismattmann** [comment]
thank you! will review @arpitaagrawal I also finished off Omim and Uniprot

File: shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/ParsedResponse.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.shangridocs.services.wikipedia.Response;
```
Comment (chrismattmann) [inline shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/ParsedResponse.java]: needs ALv2 license header

File: shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/ParsedResponse.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import java.util.List;
+
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
+
```
Comment (chrismattmann) [inline shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/ParsedResponse.java]: needs some basic javadoc

File: shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/SectionInfo.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package org.shangridocs.services.wikipedia.Response;
```
Comment (chrismattmann) [inline shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/SectionInfo.java]: ALv2 header

File: shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/SubSectionResponse.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package org.shangridocs.services.wikipedia.Response;
```
Comment (chrismattmann) [inline shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/SubSectionResponse.java]: ALv2 header

File: shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/SubSectionResponse.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
+
```
Comment (chrismattmann) [inline shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/SubSectionResponse.java]: javadoc

File: shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/WikiPages.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
+package org.shangridocs.services.wikipedia.Response;
```
Comment (chrismattmann) [inline shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/WikiPages.java]: ALv2 header. Also consider using lowercase packages (Response=>response)

File: shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/WikipediaSearchResponse.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
+package org.shangridocs.services.wikipedia.Response;
```
Comment (chrismattmann) [inline shangridocs-services/src/main/java/org/shangridocs/services/wikipedia/Response/WikipediaSearchResponse.java]: ALv2 header

File: shangridocs-webcomponents/src/main/java/org/shangridocs/webcomponents/search/search-config.json [hunk 8]
```diff
@@ -11,7 +11,7 @@
 		"Wikipedia" : {
 			"restURL" : "services/wikipedia/search",
 			"set":true
-		}
+		},
```
Comment (chrismattmann) [inline shangridocs-webcomponents/src/main/java/org/shangridocs/webcomponents/search/search-config.json]: can you separate this out into a separate pull request? @arpitaagrawal

**Comment by arpitaagrawal** [comment]
@chrismattmann : Incorporated the comments for Wikipedia API change and committed the same.

**Comment by chrismattmann** [comment]
thanks @arpitaagrawal merged!
